common/src/main/scala/uk/ac/wellcome/platform/archive/common/bagit/models/BagFetch.scala

```diff
@@ -58,12 +60,14 @@ object FetchContents {
         }
 
       }
+
+    BagFetch(entries)
   }
 
-  def write(entries: Seq[FetchEntry]): String =
+  def write(entries: Seq[BagFetchEntry]): String =
```

Comment:
minor: It would be nice if these methods were symmetric – could this take a `BagFetch` instead?